features/netsocket/cellular/generic_modem_driver/PPPCellularInterface.cpp

```diff
@@ -398,7 +404,11 @@ bool PPPCellularInterface::nwk_registration(uint8_t nwk_type)
 
 bool PPPCellularInterface::is_connected()
 {
-    return dev_info.ppp_connection_up;
+    if (_connect_status == NSAPI_STATUS_GLOBAL_UP || _connect_status == NSAPI_STATUS_LOCAL_UP) {
```

Comment:
I have a slight preference for

    return _connect_status == NSAPI_STATUS_GLOBAL_UP || _connect_status == NSAPI_STATUS_LOCAL_UP

Maybe with `()`.

I think `if (x) { return true; } else { return false; }` is kind of a tautology.